ntp.go

```diff
@@ -139,12 +144,23 @@ type Response struct {
 	Leap           LeapIndicator // server's leap second indicator; see RFC 5905
 }
 
+type QOption struct {
+	Timeout time.Duration // defaults to defaultTimeout
```

Comment:
This comment should be

`// defaults to 5 seconds`

since `defaultTimeout` is private and will not be visible in the documentation.

(I think it's fine and appropriate that defaultTimeout is private, by the way.)